modules/katamari/src/main/ts/ephox/katamari/api/Main.ts

```diff
@@ -22,6 +22,7 @@ import * as Resolve from './Resolve';
 import { Result } from './Result';
 import * as ResultInstances from './ResultInstances';
 import * as Results from './Results';
+import * as Regex from './Regex';
```

Comment:
Looks like CI is failing because this is out of order. Once that is fixed we should be able to merge this.
```
18:19:47  @ephox/katamari:   25:1  error  `./Regex` import should occur before import of `./Resolve`  import/order
```